### PR TITLE
Avatar Shopping MVP Branch into Master

### PR DESCRIPTION
**Pull request by misslivirose**
This PR has largely been tested in production and is now being merged into master. It will undergo a full test pass externally once merged.

**Test Plan (Internal)**
1. Set up wallet if it is not in a usable state
2. Go to hifi://dev-content and enter A&J
3. Attempt to try on the beanie, adjusting, removing via HMD mode
4. Go to the middle checkout zone and follow checkout process (HMD)
5. Repeat 3&4 with desktop mode (no adjustment, PR still outstanding)

### CODE REVIEW

File: DomainContent/AvatarStore/CheckoutZone/checkoutScanner.js [hunk 1]
```diff
@@ -8,58 +8,303 @@
...
-    
+    var CHECKOUT_INTERVAL_MS = 500;
+    var LIST_NAME = "highlightList3";
+    var TRANSFORMS_SETTINGS = 'io.highfidelity.avatarStore.checkOut.tranforms';
+    var SEARCH_RADIUS = 2;
+    var SEARCH_RADIUS_MINI = 1;
+    var HIGHLIGHT = Script.require('../ExternalOutlineConfig.js');
+    var SCANNED_LOCAL_ROTATION = Quat.fromVec3Degrees({ x: 10, y: 140, z: 0 });
+    var SCANNED_LOCAL_HEIGHT = 0.29;
+    var MAKING_SURE_INTERVAL = 100; // Milliseconds
```
Comment (elisa-lj11) [inline DomainContent/AvatarStore/CheckoutZone/checkoutScanner.js]: Can this be renamed to something more descriptive?

File: DomainContent/AvatarStore/CheckoutZone/checkoutScanner.js [hunk 2]
```diff
@@ -8,58 +8,303 @@
...
+    var STOP_MAKING_SURE_TIMEOUT = 5000; // Milliseconds
```
Comment (elisa-lj11) [inline DomainContent/AvatarStore/CheckoutZone/checkoutScanner.js]: This one too?

File: DomainContent/AvatarStore/CheckoutZone/checkoutZone.js [hunk 3]
```diff
@@ -10,39 +10,63 @@
...
     var APP_ICON = "https://hifi-content.s3.amazonaws.com/rebecca/CheckoutZone/shoppingCart.svg";
+    var OVERLAY_ROTATIONAL_OFFSET = { x: 10, y: 140, z: 0 };
+    
     var TABLET = Tablet.getTablet("com.highfidelity.interface.tablet.system");
-    var TABLET_ROTATIONAL_OFFSET = { x: 10, y: 240, z: 0 };
+    var TABLET_ROTATIONAL_OFFSET = { x: 10, y: 220, z: 0 };
+    var MARKETPLACE_WALLET_QML_PATH = Script.resourcesPath() + "qml/hifi/commerce/wallet/Wallet.qml";
+    // Milliseconds
+    var MAKING_SURE_INTERVAL = 100;
+    var SHORTER_STOP_INTERVAL = 1000;
```
Comment (elisa-lj11) [inline DomainContent/AvatarStore/CheckoutZone/checkoutZone.js]: Same here for above two lines

**Comment by dback2** [comment]
Tests Run:

Set up wallet if it is not in a usable state 
Go to hifi://dev-content and enter A&J
Attempt to try on the beanie, adjusting, removing via HMD mode **PASS**
Go to the middle checkout zone and follow checkout process (HMD) **PASS - but no HFC so could not buy**
Repeat 3&4 with desktop mode (no adjustment, PR still outstanding)

**Pull request merged**